07_cmd_lab/cmdlab.txt

```diff
@@ -1,13 +1,23 @@
-Intro To Command Line Lab.
+Intro To Command Line Lab - Amir Meiron
 Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
   - your current shell?
+  echo $0 > bash
```

Comment:
administrative note: 
the > symbol has a very spacial meaning, as you'll learn soon.. 
I suggest to use the "#" symbol it means a comment 
(like // in many languages).

for example:
``` echo $0  # here you can put notes, and they are ignored  ```